packages/auth/package.json

```diff
@@ -44,6 +44,12 @@
   ],
   "license": "MIT",
   "oclif": {
+    "additionalHelpFlags": [
+      "-h"
+    ],
+    "additionalVersionFlags": [
+      "-v"
+    ],
     "commands": "./lib/commands",
     "bin": "heroku",
     "devPlugins": [
```

Comment:
Delete this - oclif/plugin-help is included in core

Reply:
Nice catch.